src/momento/aio/simple_cache_client.py

```diff
@@ -273,6 +299,105 @@ async def get(self, cache_name: str, key: str) -> CacheGetResponse:
         return await self._data_client.get(cache_name, key)
 
 
+class SimpleCacheClientIncubating(SimpleCacheClient):
+    def __init__(
+        self,
+        auth_token: str,
+        default_ttl_seconds: int,
+        data_client_operation_timeout_ms: Optional[int],
+    ):
+        warnings.warn(INCUBATING_WARNING_MSG)
+        super().__init__(
+            auth_token, default_ttl_seconds, data_client_operation_timeout_ms
+        )
+
+    async def dictionary_set(
+        self,
+        cache_name: str,
+        dictionary_name: str,
+        dictionary: Dictionary,
+    ) -> CacheDictionarySetResponse:
+        """Store dictionary items (key-value pairs) in the cache.
+
+        Inserts items from `dictionary` into a dictionary `dictionary_name`.
+        Updates (overwrites) values if the key already exists.
+
+        Args:
+            cache_name (str): Name of the cache to store the dictionary in.
+            dictionary_name (str): The name of the dictionary in the cache.
+            dictionary (Dictionary): The items (key-value pairs) to be stored.
+
+        Returns:
+            CacheDictionarySetResponse: data stored in the cache
+        """
+        dictionary_get_response = await self.get(cache_name, dictionary_name)
+        cached_dictionary = {}
+        if dictionary_get_response.status() == CacheGetStatus.HIT:
+            cached_dictionary = cast(
+                Dictionary,
+                pickle.loads(cast(bytes, dictionary_get_response.value_as_bytes())),
+            )
+
+        cached_dictionary.update(convert_dict_values_to_bytes(dictionary))
+
+        set_response = await self.set(
+            cache_name, dictionary_name, pickle.dumps(cached_dictionary)
+        )
+        return CacheDictionarySetResponse(
+            key=set_response._key, value=_deserialize_stored_hash(set_response._value)
+        )
```

Comment:
nit: same argument complaint, but both different and intensified by the presence of `kwargs`

Reply:
Black code formatter again

Reply:
that is one petard by which i tire of being hoist